app/models/ability.rb

```diff
@@ -20,9 +20,5 @@ def initialize(user)
       can [:index], Vote
       can [:new, :create], VotePost
     end
-
-    user.permissions.each do |permission|
-      can permission.action.to_sym, permission.subject
-    end
```

Comment:
The quick solution would be to just change their inheritance to `Admin::BaseController`

Or the rewrite I did ;)